### DISCUSSION TITLE
Browserless with Chrome Extensions

### DISCUSSION THREAD

**Discussion by janwilmake** [browserless/chrome#3766] [General]
I am looking to run https://github.com/ishan0102/vimGPT using browserless.io for its scalability. This requires me to run chromium with [Vimium](https://vimium.github.io/).

I saw in your [blog](https://www.browserless.io/blog/2023/12/06/browserless-2-0/) that you plan on supporting extensions. What is your timeline on this?

**Comment by joelgriffith** [comment]
Hey there! We're actively working on it at the moment, and can share timeframes once we're closer. You can also (alternatively) use Docker Volumes to mount any extension files you'd like and then reference them with launch arguments. That likely the fastest way to get external extensions set up an supported right now

**Comment by Ballz-commits** [comment]
Can u explain a bit on how to do this ? .. While connecting via cdp , i don't think we can load extensions @joelgriffith
